molecule/config.py

```diff
@@ -392,7 +392,7 @@ def _get_defaults(self):
             },
             'scenario': {
                 'name':
-                'default',
+                os.path.split(os.path.dirname(self.molecule_file))[-1],
```

Comment:
We should add some tests for changes here since this is core functionality.

Reply:
@lwm please, please please... mo mandadatory options on cli! So many reasons:

This would prevent us from reusing the same file using symlinks. That was the point of not having to define a scenario name. 

Making the -s mandatory would be a serious UX downgrade and mostly a bad design. A tool should almost never have mandatory arguments, as in this case there are support to be arguments.

Still, you touched an interesting aspect: why do we really need to have an implicit "default" scenario in the first place? To me I would say that default behavior should be to run all scenarios found and to be able to optionally add `-s` param to limit it to a specific scenario (or list, or glob). It would not be really bad to be able to `-s foo*:bar`.

I think that we could look at tox cli options an lear from them: by default it runs all  environments defined in env_list but user can also tell it to run a specific one(or list) via `tox -e foo` or `TOX_ENV` variable. I would really find tox as "sucks!" if it woudl always force me to mention the -e param.

Regarding testing, I guess I could change the tests in order to validate that the directory name ends-up as being used as implicit scenario name.

For the other changes I think is better to leave them to their own tickets and separated PRs as big PRs are hard to review and merge, and more likely to break things.

Reply:
+1 to no mandadatory options on cli.